lib/posix/Kconfig

```diff
@@ -106,6 +110,39 @@ endif # FILE_SYSTEM
 
 if POSIX_API
 
+config POSIX_CLOCK_SNTP_INIT
+	bool "Initialize system clock using SNTP on application startup"
+	help
+	  Perform an SNTP request over networking to get and absolute
+	  wall clock time, and initialize system time from it, so
+	  functions like time(), gettimeofday(), etc. returned the
+	  correct absolute time (no just time since system startup).
+	  Requires networking.
```

Comment:
BTW, as this requires networking, shouldn't we have `depends on NETWORKING && SNTP` here?

Reply:
> shouldn't we have depends on NETWORKING && SNTP here?

Ack, added.